com.wamas.ide.launching/src/com/wamas/ide/launching/generator/DependencyResolver.xtend

```diff
@@ -46,26 +46,21 @@ class DependencyResolver {
 		}
 	}
 
-	static def findDependencies(LaunchConfig config, boolean addAll) {
+	static def findDependencies(LaunchConfig config, WorkspaceProductModel model, boolean addAll) {
```

Comment:
I would prefer that this method is not changed, even if this means loading the product model twice - this is API and used by external tooling already. Alternatively if you think this would be a major issue, please provide a second method with the original signature which delegates to this method (passing null is ok in this case).

Reply:
I think it is fine to load the product model twice. I will revert the changes on DependencyResolver.